src/rules/no-passthrough-getters.ts

```
@@ -0,0 +1,145 @@
+        // Skip if the getter has decorators (like @Memoize)
+        if (node.decorators && node.decorators.length > 0) {
+          return;
+        }
+
+        const methodBody = node.value.body;
+        if (!methodBody) {
+          return;
+        }
+
+        // Check if the getter body is a simple return statement
+        if (methodBody.body.length === 1 && methodBody.body[0].type === 'ReturnStatement') {
```

ESLint

[error] Replace `methodBody.body.length·===·1·&&·methodBody.body[0].type·===·'ReturnStatement'` with `⏎··········methodBody.body.length·===·1·&&⏎··········methodBody.body[0].type·===·'ReturnStatement'⏎········`

(prettier/prettier)